### PR TITLE
Views: Header Revamp

### PR DESCRIPTION
**Pull request by gblakeman**
Rebuild the `HeaderBar` component with the new design and state variations (default, views, saved, controls on/of, etc.).

Screen Shot 2022-01-31 at 10 28 12 AM

Screen Recording 2022-01-31 at 10 30 42 AM

### CODE REVIEW

**Comment by joeyyang** [comment]
found a UI bug / unexpected interaction - are these icons supposed to be disabled / trigger some menu or action? here were my steps to reproduce:
- select a frame with content elements
- click header controls

expected: clicking the 'filter' or '...' controls triggers another UI change
actual: icon lights up blue, but no other changes

realish-v2:header-revamp

**Review by joeyyang — COMMENTED** [review]
Looks good code-wise mostly, found a UI bug / unexpected interaction, could you clarify?

**Review by joeyyang — APPROVED** [review]
Minor readability nits but otherwise looks good!

File: src/views/App.svelte [hunk 1]
```diff
@@ -77,10 +79,8 @@
 
 <!-- core layout -->
 <div
-  class="
-    {(isComponentPreview || (items && items.length)) ? '' : 'blank-state-visible'}
-    {(process.env.BETA && betaLocked) ? ' locked-state-visible' : ''}
-    {isComponentPreview ? ' component-preview-state-visible' : ''}"
+  class="{(isComponentPreview || (items && items.length)) ? '' : 'blank-state-visible'}
```
[the remark this answers is not in the document]
Comment (gblakeman) [reply src/views/App.svelte:82]: Great suggestion – this code goes away in the next PR though, so going to leave it be… 😂

**Pull request merged**